### PR TITLE
[Merged by Bors] - Blend Modes and Fog Release Notes for 0.10

### PR DESCRIPTION
**Pull request by coreh**
This PR adds release note write ups for **Fog** and **Blend Modes**.

Based on #546.

Some remarks/comments:

- Proofreading/grammar/spelling corrections appreciated 😄
- For the PR author, is it customary to just include the PR committer (i.e. me) or do I also include the reviewers who provided code suggestions?
- Let me know if the sections are too detailed
- Not sure if the SVG plots will render correctly, will probably depend on the markdown engine
- Can someone help me with the right settings for converting the demo video? I keep getting much larger files than the existing ones. Do y'all have a ready-made `ffmpeg` command-line snippet I can use? 😄

### CODE REVIEW

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 1]
```diff
@@ -587,6 +587,152 @@ query.iter().for_each(|mut component| {
...
+
+## Distance and Atmospheric Fog
+
+<div class="release-feature-authors">author: @coreh</div>
+
+Bevy can now render distance and atmospheric fog effects, bringing a heightened sense of _depth_ and _ambiance_ to your scenes by making objects appear dimmer the further away they are from view.
+
+<figure>
+<img src="fog.png">
+<figcaption>The new <code>fog</code> example, showcasing different fog modes and parameters.</figcaption>
```
Comment (LiamGallagher737) [inline content/news/2023-03-04-bevy-0.10/index.md:627]:
```suggestion
<figcaption>The new <code>fog</code> example, showcases different fog modes and parameters.</figcaption>
```

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 2]
```diff
@@ -587,6 +587,152 @@ query.iter().for_each(|mut component| {
...
+</svg>
+</figure>
+
+Additionally, a more sophisticated `FogFalloff::Atmospheric` mode is available which provides *more physically accurate* results by taking light `extinction` and `inscattering` into account separately.
+
+`DirectionalLight` influence is also supported for all fog modes via the `directional_light_color` and `directional_light_exponent` parameters, mimicking the light dispersion effect seen on sunny outdoor environments.
+
+<figure>
+<img src="atmospheric-fog.png">
+<figcaption>The new <code>atmospheric_fog</code> example, showcasing a terrain with atmospheric fog and directional light influence.</figcaption>
```
Comment (LiamGallagher737) [inline content/news/2023-03-04-bevy-0.10/index.md:719]:
```suggestion
<figcaption>The new <code>atmospheric_fog</code> example, showcases a terrain with atmospheric fog and directional light influence.</figcaption>
```

**Review by LiamGallagher737 — COMMENTED** [review]
I'm a bit confused with all the `<figure>` and `<figcaption>` blocks, could you elaborate on why you are using these as for me on GitHub mobile the markdown render looks the same had they not been there.

**Comment by LiamGallagher737** [comment]
To answer your question on whether your SVG plots are viewable, I cannot see them in GitHub mobile nor can I see them on the web version of GitHub.

**Comment by coreh** [comment]
They're supposed to look like this: 

image

I guess GitHub restricts SVG for security reasons, blog engines typically assume first-party content and are less restrictive... I know the SVG plots work fine on generated rust documentation, so I'm hopeful it will work on the site? Haven't tried building it locally yet

**Comment by coreh** [comment]
> I'm a bit confused with all the <figure> and <figcaption> blocks, could you elaborate on why you are using these as for me on GitHub mobile the markdown render looks the same had they not been there.

Depending on the CSS it might center or indent the content, or put a different/smaller font for the caption, or a background. If the style doesn't do anything with it, I can take it out and just wrap it in `_`, but maybe it's worth it to have the semantic markup in place in case the site theme evolves in the future?

File: content/news/2023-03-04-bevy-0.10/index.md [hunk 3]
```diff
@@ -587,6 +587,152 @@ query.iter().for_each(|mut component| {
 
 These abstractions were introduced in [#6404](https://github.com/bevyengine/bevy/pull/6404), [#7381](https://github.com/bevyengine/bevy/pull/7381) and [#7568](https://github.com/bevyengine/bevy/pull/7568).
 
+## `StandardMaterial` Blend Modes
```
Comment (cart) [inline content/news/2023-03-04-bevy-0.10/index.md:590]: Sadly "code" formatting in headers is really bad in our CSS right now. Can you remove it for now?

**Comment by LiamGallagher737** [comment]
Would you be able to move the svgs into their own files? That way GitHub should be able to display them and it would also tidy up the main file. As a side note those graphs are a really nice touch.

**Comment by cart** [comment]
> For the PR author, is it customary to just include the PR committer (i.e. me) or do I also include the reviewers who provided code suggestions?

Generally just the author / committer, unless someone provided significant contributions (either code or designs) 

> Let  me know if the sections are too detailed

For now this is great. I might do a global trimming pass across all sections later if we're too long.

> Not sure if the SVG plots will render correctly, will probably depend on the markdown engine
   
It renders in firefox for me when zola renders the template.

> Can someone help me with the right settings for converting the demo video? I keep getting much larger files than the existing ones. Do y'all have a ready-made ffmpeg command-line snippet I can use? smile

I just use Handbrake to re-encode. If you send files my way I can do it.

**Comment by coreh** [comment]
> Would you be able to move the svgs into their own files? That way GitHub should be able to display them and it would also tidy up the main file.

As a last resort, we can do that. The main reason I didn't do it is because I'm using `currentColor` for the text and non-colored lines, so that the end result fits nicely with whatever dark/light theme the rust documentation and bevy website might be using.

> As a side note those graphs are a really nice touch.

☺️

**Pull request closed without merging**